pkg/rke2/rke2.go

```diff
@@ -91,7 +91,6 @@ func setup(clx *cli.Context, cfg Config) error {
 
 	agentManifestsDir := filepath.Join(dataDir, "agent", config.DefaultPodManifestPath)
 	agentImagesDir := filepath.Join(dataDir, "agent", "images")
-	cisMode := clx.String("profile") != ""
```

Comment:
Ugh. I removed this line in a previous PR since it's being set above. Not sure how that got back.